content/blog/esc-db-secrets-rotation-launch/index.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Introducing Automated Database Credential Rotation for PostgreSQL, MySQL, and Snowflake in Pulumi ESC"
+date: 2025-04-30 # Adjust T00:00:00-00:00 as needed for timezone/exact time
+allow_long_title: true
+meta_desc: "Pulumi ESC now automates the rotation of database credentials for PostgreSQL, MySQL (on AWS), and Snowflake, enhancing security and reducing operational burden."
+meta_image: meta.png
+authors:
+- sean-yeh
+- claire-gaestel
+- arun-loganathan
+tags:
+- esc
+- secrets
+- features
+- rotation
+- database
+- security
+---
+
+Securing access to critical data stores is paramount in today's cloud-native world. Yet, managing database credentials often involves static, long-lived passwords – a significant security blind spot. These static secrets, frequently embedded in application configurations or accessible to multiple team members, represent a prime target for attackers. Manually rotating these credentials is a cumbersome, error-prone task that’s often neglected, leaving databases vulnerable for extended periods. Building on our commitment to robust secrets management, we are excited to launch **Automated Database Credential Rotation** for **PostgreSQL, MySQL, and Snowflake** in [Pulumi ESC!](/product/esc)
+
+<!--more-->
+
+## The Persistent Challenge of Static Database Secrets
+
+Relying on static database credentials introduces substantial risks, which automated rotation helps mitigate:
+
+*   **Security Vulnerabilities & Exposure:** Static credentials, if compromised through leaks, phishing, or unauthorized access, provide long-term access to attackers. Automated rotation significantly shrinks this window of opportunity.
+*   **Operational & Compliance Burdens:** Manually rotating database credentials is complex, error-prone, and requires careful coordination to avoid downtime. It also makes demonstrating compliance with regulations like SOC 2, GDPR, or HIPAA (which often mandate rotation) difficult and hard to audit consistently.
```

Comment:
Last sentence is hard to read, IMO 
```suggestion
*   **Operational & Compliance Burdens:** Manually rotating database credentials is complex, error-prone, and requires careful coordination to avoid downtime. These manual rotations are hard to audit and demonstrate compliance with regulations like SOC 2, GDPR, or HIPAA (which often mandate rotation).
```